pkgs/games/lugaru/default.nix

```diff
@@ -0,0 +1,29 @@
+{ fetchurl, stdenv, cmake, openal, pkgconfig, libogg,
+  libvorbis, SDL2, makeWrapper, libpng, libjpeg_turbo, libGLU }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+
+  pname = "lugaru";
+  version = "1.2";
+
+  src = fetchurl {
```

Comment:
please use `fetchfromgitlab`, if there's no specific reason not to.

Reply:
done . i don't know there is such thing for gitlab ...